doc/data/frequency-plans.yml

```diff
@@ -0,0 +1,105 @@
+- id: EU_863_870
+  name: Europe 863-870 MHz
+  description: Default frequency plan for Europe
+  base-frequency: 868
+  file: EU_863_870.yml
+
+- id: EU_863_870_TTN
+  name: Europe 863-870 MHz (TTN)
+  description: TTN Community Network frequency plan for Europe, using SF9 for Rx2
```

Comment:
```suggestion
  description: TTN Community Network frequency plan for Europe, using SF9 instead of SF12 for Rx2
```

Reply:
This kind of confusion needs to be avoided. This file is just a copy and any changes here will be overwritten.

Reply:
```suggestion
  description: TTN Community Network frequency plan for Europe, using SF9 for RX2
```